### PR TITLE
Feature/Auto Complete component

### PR DESCRIPTION
**Pull request by Kelvin-GohKengBoon**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots (if applicable)

Please add appropriate screenshots.

### CODE REVIEW

**Review by naimdanialjohar — APPROVED** [review]

File: src/AutoComplete/style.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@import '../assets/style/form-item.css';
```
Comment (AladdinNullptr) [inline src/AutoComplete/style.css:1]: only related to the form. Should be imported in the story

Comment (AladdinNullptr) [reply src/AutoComplete/style.css:1]: Are they styles below also depending on a form? if yes please remove this dependency

Comment (AladdinNullptr) [reply src/AutoComplete/style.css:1]: auto complete default style import missing

File: src/AutoComplete/style.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@import 'antd/lib/style/default.css';
```
Comment (AladdinNullptr) [inline src/AutoComplete/style.css:2]: auto complete default style import missing

File: src/AutoComplete/style.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+@import 'antd/lib/select/style/index.css';
```
Comment (AladdinNullptr) [inline src/AutoComplete/style.css:3]: can be replaced with `@import '../Select/style.css';`

File: src/AutoComplete/style.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+@import 'antd/lib/empty/style/index.css';
+@import '../Input/style.css';
```
Comment (AladdinNullptr) [inline src/AutoComplete/style.css:5]: Why do you need the styles? I don't see them in `css.js`

File: src/AutoComplete/style.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+@import '../Select/style.css';
```
Comment (AladdinNullptr) [inline src/AutoComplete/style.css:6]: remove. The final import order must be the same as `css.js`

**Review by AladdinNullptr — APPROVED** [review]

**Pull request merged**